syntax = "proto3";

package android.incremental.metadata;

message BindPoint {
    int32 storage_id = 1;
    string source_subdir = 2;
    string dest_path = 3;
}

message DataLoader {
    string package_name = 1;
    string arguments = 2;
}

message Storage {
    int32 id = 1;
}

message Mount {
    Storage storage = 1;
    DataLoader loader = 2;
}
